misc/make-deps.sh

```diff
@@ -84,9 +89,8 @@ echo "done running 'go get -v -t -d ./...'"
 [ -e "$GOBIN/mgmt" ] && rm -f "$GOBIN/mgmt"	# the `go get` version has no -X
 # vet is built-in in go 1.6 - we check for go vet command
 go vet 1> /dev/null 2>&1
-ret=$?
-if [[ $ret != 0 ]]; then
-	go get golang.org/x/tools/cmd/vet      # add in `go vet` for travis
+if [[ $? != 0 ]]; then
+	go get github.com/golang/go/tree/master/src/cmd/vet      # add in `go vet` for travis
```

Comment:
why did you make this change?

Reply:
Because if I didn't I get the following error message:

```
package golang.org/x/tools/cmd/vet: cannot find package "golang.org/x/tools/cmd/vet" in any of:
	/usr/local/go/src/golang.org/x/tools/cmd/vet (from $GOROOT)
	/home/oliver/gotest/src/golang.org/x/tools/cmd/vet (from $GOPATH)
```

I'll remove it from the PR and let you sort it out :)